_config.yml

```diff
@@ -5,7 +5,7 @@ description: "A collection of tips and advice for streamers by the StreamSounds
 logo: 'assets/images/logo.png'
 favicon: 'assets/images/logo.png'
 baseurl: ''
-google_analytics: ''
+#google_analytics: ''
```

Comment:
If we aren't using this, then let's remove it completely.

Reply:
I will wait and do that in another branch, along with discuss, mailchimp and any other properties we aren't using.